cardano-node/src/Cardano/CLI/Key.hs

```diff
@@ -1,23 +1,27 @@
 {-# LANGUAGE CPP                        #-}
 {-# LANGUAGE GeneralisedNewtypeDeriving #-}
-{-# LANGUAGE RecordWildCards            #-}
-{-# LANGUAGE ViewPatterns               #-}
+{-# LANGUAGE LambdaCase                 #-}
 {-# OPTIONS_GHC -Wno-all-missed-specialisations #-}
 
 module Cardano.CLI.Key
   ( SigningKeyFile(..)
   , NewSigningKeyFile(..)
   , VerificationKeyFile(..)
   , NewVerificationKeyFile(..)
-  , prettySigningKeyPub
+  , prettyPublicKey
   , readSigningKey
   , readVerificationKey
+  , PasswordRequirement(..)
```

Comment:
Did we decide on how to arrange exports in the style doc?

Reply:
I've added grouping to underscore the intent -- within each group the types precede functions.

Reply:
A relevant discussion in the style document: https://github.com/input-output-hk/cardano-node/pull/164#discussion_r321786601